capa/render/result_document.py

```diff
@@ -73,7 +74,7 @@ class Analysis(Model):
     arch: str
     os: str
     extractor: str
-    rules: Tuple[str, ...]
+    rules: Tuple[Path, ...]
```

Comment:
we may want to keep string here depending on the effect on the JSON output (my comment above is related)